plutus-scb/plutus-scb.cabal

```diff
@@ -144,6 +160,7 @@ library
         uuid -any,
         vector -any,
         warp -any,
+        Win32-network -any,
```

Comment:
What is this needed for?

Reply:
According to this recent blogpost, it might not be necessary: https://crypto.bi/tape/cardano-node/

Reply:
The `Win32-network` library abstracts over POSIX and Win32 socket operations (in our specific use-case it makes asynchronous socket operations work on windows - there is a longer story here, but bottom line, the socket code will probably not work properly on windows). It should compile on bot windows and linux, selecting the proper primitives in both cases.

The blog post concludes by giving the solution of upgrading to latest Ubuntu, and using the default configurations which include the `Win32-network` library.